config/detekt-baseline.xml

```diff
@@ -48,9 +48,6 @@
     <ID>LongParameterList:GeckoWebExtension.kt$GeckoWebExtension$( id: String, url: String, val runtime: GeckoRuntime, allowContentMessaging: Boolean = true, supportActions: Boolean = false, val nativeExtension: GeckoNativeWebExtension = GeckoNativeWebExtension( url, id, createWebExtensionFlags(allowContentMessaging), runtime.webExtensionController ), private val connectedPorts: MutableMap&lt;PortId, Port&gt; = mutableMapOf() )</ID>
     <ID>LongParameterList:ManifestUpdateFeature.kt$ManifestUpdateFeature$( private val applicationContext: Context, private val sessionManager: SessionManager, private val shortcutManager: WebAppShortcutManager, private val storage: ManifestStorage, private val sessionId: String, private var initialManifest: WebAppManifest )</ID>
     <ID>LongParameterList:MozillaSocorroService.kt$MozillaSocorroService$( private val applicationContext: Context, private val appName: String, private val appId: String = MOZILLA_PRODUCT_ID, private val version: String = BuildConfig.MOZILLA_VERSION, private val buildId: String = BuildConfig.MOZ_APP_BUILDID, private val vendor: String = BuildConfig.MOZ_APP_VENDOR, private var serverUrl: String? = null, private var versionName: String = DEFAULT_VERSION_NAME, private val releaseChannel: String = BuildConfig.MOZ_UPDATE_CHANNEL )</ID>
-    <ID>LongParameterList:P2PFeature.kt$P2PFeature$( private val view: P2PView, private val store: BrowserStore, private val engine: Engine, private val connectionProvider: () -&gt; NearbyConnection, private val tabsUseCases: TabsUseCases, private val sessionUseCases: SessionUseCases, private val sessionManager: SessionManager, override val onNeedToRequestPermissions: OnNeedToRequestPermissions, private val onClose: (() -&gt; Unit) )</ID>
-    <ID>LongParameterList:P2PIntegration.kt$P2PIntegration$( store: BrowserStore, engine: Engine, private val view: P2PView, thunk: () -&gt; NearbyConnection, tabsUseCases: TabsUseCases, sessionManager: SessionManager, sessionUseCases: SessionUseCases, onNeedToRequestPermissions: OnNeedToRequestPermissions )</ID>
```

Comment:
Wasn't sure if I needed to manually remove this.

Reply:
I think it should be OK